css/style.css

```diff
@@ -0,0 +1,82 @@
+html {
+  box-sizing: border-box;
+}
+
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: sans-serif;
+  margin: 0;
+  background-color: white;
+}
```

Comment:
You should use the font family as the Figma design you selected, try to click on inspect in Figma to see what is the right font family for every single text element.

- [ ]  Fix font-family for your elements in your design as you can find in the Figma design you selected.